api/external/infra_proxy/infra_proxy.proto

```diff
@@ -148,7 +148,24 @@ service InfraProxy {
     option (google.api.http).get = "/api/v0/infra/servers/{server_id}/orgs/{org_id}/clients/{name}";
     option (chef.automate.api.iam.policy).resource = "infra:servers:{server_id}:orgs:{org_id}:clients";
     option (chef.automate.api.iam.policy).action = "infra:infraServers:get";
-  }
+  };
+  rpc CreateClient (infra_proxy.request.CreateClient) returns (infra_proxy.response.CreateClient) {
+    option (google.api.http).post = "/api/v0/infra/servers/{server_id}/orgs/{org_id}/clients";
+    option (google.api.http).body = "*";
+    option (chef.automate.api.iam.policy).resource = "infra:servers:{server_id}:orgs:{org_id}:clients";
+    option (chef.automate.api.iam.policy).action = "infra:infraServers:update";
+  };
+  rpc DeleteClient (infra_proxy.request.Client) returns (infra_proxy.response.Client) {
+    option (google.api.http).delete = "/api/v0/infra/servers/{server_id}/orgs/{org_id}/clients/{name}";
+    option (chef.automate.api.iam.policy).resource = "infra:servers:{server_id}:orgs:{org_id}:clients";
+    option (chef.automate.api.iam.policy).action = "infra:infraServers:update";
```

Comment:
check the other proto file, in case of get/update/delete this policy has differerent way to handle. 
option (chef.automate.api.iam.policy).action = "infra:infraServers:delete";

Reply:
@punitmundra, here we handled it differently, in all action only get & update for all third-level objects like `chef server >> chef organizations >> cookbooks| roles| clients | data bags` etc.

Please check the detailed discussion in PR https://github.com/chef/automate/pull/3550

Reply:
fine